design/baremetal-operator/early-cleaning.md

```diff
@@ -0,0 +1,128 @@
+<!--
+ This work is licensed under a Creative Commons Attribution 3.0
+ Unported License.
+
+ http://creativecommons.org/licenses/by/3.0/legalcode
+-->
+
+# Early out-of-band cleaning
```

Comment:
Can I suggest we not mention cleaning up-front? It tends to confuse people, and ultimately is just an implementation detail for how we go about resetting the BMC through Ironic.

Reply:
done